### PR TITLE
Luxury Bluespace Capsule & More

### PR DESCRIPTION
**Pull request by tigercat2000**
tgstation/tgstation#27881

There is now a second bluespace shelter capsule, which is much more
fancy & expensive. Note, this still does not make them generally available.

Changes Summary:
 - Added "Shelter Beta" - It comes with a built in kitchen, toilet, shower, three
 course meal, and an artifact:tm:! There are plenty of secrets to
 behold...
 - Added black carpeting. This can be created by the personal crafting
 menu and having any crayon + a carpet with you.
 - Added fancy wooden and fancy black wooden tables. These are craftable
 by hitting wooden table parts with regular and black carpet,
 respectively.
 - You can now unanchor, deconstruct and color shower curtains. Use any kind of
 crayon to color them, and screwdriver to unanchor them. Once they are
 unanchored, you may use wirecutters to change them back into materials-
 All of which are used in a new crafting recipe for creating them.
 - Display cases moderately refactored.
 - Added survival pod windows/windoors for the new luxury pod.

🆑
add: Luxury versions of the bluespace shelter capsule! Currently not obtainable.
add: Black Carpet can now be crafted using a stack of carpet and a crayon.
add: Black fancy tables can now be crafted using Black Carpet.
add: Shower curtains can now be recoloured with crayons, unscrewed from the floor, disassembled with wire cutters, and reassembled using cloth, plastic, and a metal rod.
/🆑

### CODE REVIEW

File: code/modules/mining/shelters.dm [hunk 1]
```diff
@@ -40,3 +40,17 @@
 /datum/map_template/shelter/alpha/New()
 	. = ..()
 	whitelisted_turfs = typecacheof(/turf/simulated/mineral)
+
+/datum/map_template/shelter/beta
+	name = "Shelter Beta"
+	shelter_id = "shelter_beta"
+	description = "An extremly luxurious shelter, containing all \
```
Comment (SamHPurp) [inline code/modules/mining/shelters.dm]: extremely

File: code/game/objects/structures/watercloset.dm [hunk 2]
```diff
@@ -120,6 +120,15 @@
 		to_chat(user, "<span class='notice'>You carefully place [I] into the cistern.</span>")
 		return
 
+/obj/structure/toilet/secret
+	var/secret_type = null
+
+/obj/structure/toilet/secret/New()
```
Comment (taukausanake) [inline code/game/objects/structures/watercloset.dm]: What is a secret toilet besides a secret to everybody?

**Comment by taukausanake** [comment]
> black carpeting
**any crayon** + a carpet

WHY DO ALL THE COLORS MAKE BLACK?

Also, why a crayon to color shower curtains? Why not paint instead? And how do you get the cloth for the curtain construction?

**Comment by tigercat2000** [comment]
> WHY DO ALL THE COLORS MAKE BLACK?

Because we don't have black crayons and I don't want to limit these to the clown

> Also, why a crayon to color shower curtains? Why not paint instead? 

Pain(t) in the ass to obtain

> And how do you get the cloth for the curtain construction?

Biogenerator

**Comment by FalseIncarnate** [comment]
Technically the mime's crayon can do black coloring, as can spraypaint.

That said, I'm fine with it being a bit easier by requiring any crayon.

Out of curiousity, what happens if you place red and black carpet next to each other? Do they smooth/connect or do they remain separate? You could make a giant checkerboard with red and black...

**Comment by tigercat2000** [comment]
@FalseIncarnate They do not smooth, mainly because their textures are completely different

**Comment by tigercat2000** [comment]
Updated and added the lusty maid cutout, will update screenshot momentarily.

**Comment by tigercat2000** [comment]
Updated with new picture:

**Comment by Fox-McCloud** [comment]
👌

**Pull request merged**